cms/djangoapps/contentstore/views/transcripts_ajax.py

```diff
@@ -119,16 +121,24 @@ def upload_transcripts(request):
     if video_list:
         sub_attr = source_subs_name
         try:
-            # Generate and save for 1.0 speed, will create subs_sub_attr.srt.sjson subtitles file in storage.
-            generate_subs_from_source({1: sub_attr}, source_subs_ext, source_subs_filedata, item)
+            # Generate sjson subtitles from srt substitles
+            sjson_subs = generate_subs_from_source({}, source_subs_ext, source_subs_filedata, item)
```

Comment:
Why are now sending an empty dict now ?

Reply:
so that a `save_subs_to_store` function should not be executed in `generate_subs_from_source`